pkg/web/routes.go

```diff
@@ -22,6 +22,8 @@ import (
 	"errors"
 	"fmt"
 	"github.com/armory/dinghy/pkg/dinghyfile/pipebuilder"
+	log2 "github.com/armory/dinghy/pkg/log"
```

Comment:
Perhaps instead of `log2` we alias this as `customLog`?

Reply:
Changed to dinghylog